Dockerfile.dapper783451420

```diff
@@ -0,0 +1,22 @@
+FROM quay.io/submariner/shipyard-dapper-base:devel
```

Comment:
This file should not be added :)

Reply:
Done . it was due to a  `git add .` . :)